tsup.config.ts

```diff
@@ -1,11 +1,11 @@
 import { defineConfig } from "tsup";
 
 export default defineConfig({
-  entry: ["src/main"],
+  entry: ["src/main/index.ts"],
   outDir: "build",
   format: ["cjs", "esm"],
   target: "es2019",
   sourcemap: true,
   dts: true,
-  splitting: false,
```

Comment:
I realized this `splitting` attribute is actually pointless, as all of your code is included in a single file anyway haha.

Reply:
Makes sense.